README.md

```diff
@@ -248,6 +248,18 @@ As an example we use the Symfony `DateTimeNormalizer` service so we do have supp
 
 As a side note: If you happen to use [Autowiring](https://symfony.com/doc/current/service_container/autowiring.html) in your `services.yaml` you might need to set `autowire: false` too.
 
+**When namespace of Model used change**
+
+You must have migrated the data in the database because your JSON has `#type` key which contains the namespace of your used Model class.
+
+For example if we have a project that we migrate to Symfony/flex,
```

Comment:
What do you think about this (it's not related to Symfony Flex at all so I left that away):

```
Example: If we have a project that we migrate from `AppBundle` to `App`, we have the namespace `AppBundle/Entity/Bar` in our database which has to become `App/Entity/Bar` instead.
```

Reply:
it's much better, thank you ;)